CHANGELOG.md

```diff
@@ -20,10 +20,11 @@
 
 - [core] Support custom axios options for all request builders.
 - [core] Support disabling csrf token request as an option for all request builders.
+- [odata] Support unary operator `not` in the `filter()` of OData request builders.
 
 ## Improvements
 
--
+- [odata] Filter functions with boolean return types can be used directly in the `filter()` of OData request builders without `equals(true)`.
```

Comment:
Try to formulate it so that it matches the imperative case (check the examples on top of this file)